bigbluebutton-html5/imports/ui/components/muted-alert/component.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import hark from 'hark';
+import Icon from '/imports/ui/components/icon/component';
+import { styles } from './styles';
+
+const MUTE_WARNING_TIMEOUT = 4000;
+
+const propTypes = {
+  inputDeviceId: PropTypes.string.isRequired,
+};
+
+class MutedAlert extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      visible: false,
+    };
+
+    this.speechEvents = null;
+    this.timer = null;
+
+    this.resetTimer = this.resetTimer.bind(this);
+  }
+
+  componentDidMount() {
+    this._isMounted = true;
+    const { inputDeviceId } = this.props;
+
+    navigator.mediaDevices.getUserMedia({ audio: { deviceId: inputDeviceId } })
```

Comment:
Hmm. This might be problematic. Asking for another gUM might make permission prompts pop in environments where device permissions can't be remembered (ie running BBB in an iframe, folk didn't mark the remember option). Moreover, the less gUM calls, the better. AFAIK audio-manager already has a getter for the `inputStream` which you could pass down as a prop to this component (as you did with the inputDeviceId). Just have to take care not to leak it.